packages/web-app-preview/src/components/Sources/MediaVideo.vue

```diff
@@ -18,6 +25,23 @@ export default defineComponent({
       type: Boolean,
       default: true
     }
+  },
+  beforeUnmount() {
+    window.removeEventListener('resize', this.resizeVideoDimensions)
+  },
+  methods: {
```

Comment:
Please use the composition API by adding the methods and `onBeforeUnmount` in `setup`.